libs/image-editor/build.gradle

```diff
@@ -46,42 +46,38 @@ repositories {
-    implementation "androidx.constraintlayout:constraintlayout:$androidxConstraintlayoutVersion"
-    implementation "androidx.viewpager2:viewpager2:$androidxViewpager2Version"
-    implementation "com.google.android.material:material:$googleMaterialVersion"
-    implementation "androidx.navigation:navigation-fragment:$gradle.ext.navigationVersion"
-    implementation "androidx.navigation:navigation-ui:$gradle.ext.navigationVersion"
-    implementation "androidx.lifecycle:lifecycle-common:$androidxLifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-runtime:$androidxLifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-viewmodel:$androidxLifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-viewmodel-savedstate:$androidxLifecycleVersion"
-    implementation "androidx.lifecycle:lifecycle-livedata-core:$androidxLifecycleVersion"
-    implementation ("com.github.yalantis:ucrop:$uCropVersion") {
+    implementation libs.androidx.core
+    implementation libs.androidx.activity
+    implementation libs.androidx.activity.ktx
+    implementation libs.androidx.fragment
+    implementation libs.androidx.appcompat
+    implementation libs.androidx.recyclerView
+    implementation libs.androidx.constraintLayout
+    implementation libs.androidx.viewPager2
+    implementation libs.google.material
+    implementation libs.navigation.fragment
+    implementation libs.navigation.ui
+    implementation libs.androidx.lifecycle.common
+    implementation libs.androidx.lifecycle.runtime
+    implementation libs.androidx.lifecycle.viewmodel
+    implementation libs.androidx.lifecycle.viewmodel.savedstate
+    implementation libs.androidx.lifecycle.livedata.core
+    implementation (libs.ucrop) {
         exclude group: 'com.squareup.okhttp3'
         exclude group: 'androidx.core', module: 'core'
         exclude group: 'androidx.constraintlayout', module: 'constraintlayout'
         exclude group: 'androidx.appcompat', module: 'appcompat'
     }
 
-    testImplementation "org.mockito.kotlin:mockito-kotlin:$mockitoKotlinVersion"
-    testImplementation "junit:junit:$junitVersion"
-    testImplementation "org.assertj:assertj-core:$assertjVersion"
-    testImplementation("androidx.arch.core:core-testing:$androidxArchCoreVersion", {
+    testImplementation libs.mockito.kotlin
+    testImplementation libs.junit
+    testImplementation libs.assertj.core
+    testImplementation(libs.androidx.coreTesting, {
         exclude group: 'com.android.support', module: 'support-compat'
         exclude group: 'com.android.support', module: 'support-annotations'
         exclude group: 'com.android.support', module: 'support-core-utils'
     })
 
-    androidTestImplementation "androidx.test.ext:junit:$androidxTestExtJunitVersion"
-    androidTestImplementation("androidx.test.espresso:espresso-core:$androidxTestEspressoVersion") {
-        version {
-            strictly androidxTestEspressoVersion
-        }
-    }
```

Comment:
Warning (⚠️): Let's not remove this configuration unnecessarily. You could instead do the following:

```groovy
        version {
            strictly libs.versions.androidxTestEspresso.get()
        }
```

PS: Same applies everywhere else where you removed such configuration.

Reply:
fixed.

Reply:
> PS: Same applies everywhere else where you removed such configuration.

👋 @neeldoshii , just an FYI that you are still missing some `androidxTestEspresso` configurations that need reverting as well. I purposely didn't find those for you, just this one. 😊

Reply:
FYI: The usual process for `Resolve conversation` is that a reviewer resolves their own comments, not the author. This way, the reviewer can review the changes and any comments that are still open on a PR (not collapsed). Otherwise it is harder for the reviewer to know which comments are being resolved and why, without him doing the resolving.

Does that makes sense to you? 🙏

Reply:
> 👋 @neeldoshii , just an FYI that you are still missing some androidxTestEspresso configurations that need reverting as well. I purposely didn't find those for you, just this one. 😊

Fixed in [223f669](https://github.com/wordpress-mobile/WordPress-Android/pull/21029/commits/223f6699b3dd8e011af0c709f5bdffe5b7764f5c)

> FYI: The usual process for Resolve conversation is that a reviewer resolves their own comments, not the author. This way, the reviewer can review the changes and any comments that are still open on a PR (not collapsed). Otherwise it is harder for the reviewer to know which comments are being resolved and why, without him doing the resolving.

Sure! gotcha.

Reply:
Check [this](https://github.com/wordpress-mobile/WordPress-Android/pull/21029/commits/223f6699b3dd8e011af0c709f5bdffe5b7764f5c#r1672116519) comment please @neeldoshii . 😊